### PR TITLE
CapabilityStatement & Profile Notes pages update

### PR DESCRIPTION
**Pull request by dt-r**
Excluded from this PR is [FHIR-46309](https://jira.hl7.org/browse/FHIR-46309).

This PR addresses consistency and completeness across Profile Notes & CapabilityStatements including:
- add in multiple Or, multipleAnd into the profile notes params & anything else missing that is in the cap statements
- improve the explanation of _id
- add into the cap statements any search param documentation missing that is present on the profile pages
- fix the row borders on the profile page params where missing
- added to capstatements notes on medreq support the _include
- add Must Support choice option documentation into the cap statements documentation for MedicationRequest, Encounter, Procecdure

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-condition-notes.md [hunk 1]
```diff
@@ -65,13 +65,13 @@
         <td>code</td>
         <td><b>MAY</b></td>
         <td><code>token</code></td>
-        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.</td>
+        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.<br/><br/>The requester <strong>SHALL</strong> support <code>multipleOr</code>. The responder <strong>SHALL</strong> support <code>multipleOr</code>.</td>
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-condition-notes.md:68]: Should be '<strong>SHOULD</strong> support <code>multipleOr</code>'.

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md [hunk 2]
```diff
@@ -32,27 +32,27 @@
...
 
     `GET [base]/Practitioner?identifier={system|}[code]`
 
     Example:
     
       1. GET [base]/Practitioner?identifier=http://ns.electronichealth.net.au/id/hi/hpii/1.0\|8003619900015717
+      1. GET [base]/Practitioner?identifier=8003619900015717
 
-    *Implementation Notes:* Fetches a bundle containing any Practitioner resources matching the identifier ([how to search by token](http://hl7.org/fhir/R4/search.html#token))
+    *Implementation Notes:* Fetches a bundle containing any Practitioner resources matching the identifier ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id))
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-practitioner-notes.md:44]: I think this should still be how to search by token.

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md [hunk 3]
```diff
@@ -32,27 +32,27 @@
...
 
-    `GET [base]/Practitioner/[id]` or `GET [base]/Practitioner?_id=[id]`
+    `GET [base]/Practitioner?_id=[id]`
 
     Example:
     
-      1. GET [base]/Practitioner/5678
       1. GET [base]/Practitioner?_id=5678
 
     *Implementation Notes:* Returns a single Practitioner resource ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id))
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-practitioner-notes.md:58]: Missing the change to improve the explanation of _id here.

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md [hunk 4]
```diff
@@ -60,38 +60,40 @@ The following search parameters **SHALL** be supported:
...
+    or optionally `GET [base]/PractitionerRole?practitioner.identifier=[system|][code]`
 
     Example:
     
       1. GET [base]/PractitionerRole?practitioner=Practitioner/sandyson-sandy
       1. GET [base]/PractitionerRole?practitioner=Practitioner/sandyson-sandy&amp;_include=PractitionerRole:practitioner
       1. GET [base]/PractitionerRole?practitioner.identifier=http://ns.electronichealth.net.au/id/hi/hpii/1.0\|8003619900015717
 
-    *Implementation Notes:* Fetches a bundle of all PractitionerRole resources matching the specified practitioner ([how to search by reference](http://hl7.org/fhir/R4/search.html#reference))
+    *Implementation Notes:* Fetches a bundle of all PractitionerRole resources matching the specified practitioner ([how to search by reference](http://hl7.org/fhir/R4/search.html#reference)) and [how to search by token](http://hl7.org/fhir/R4/search.html#token))
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md:80]: Remove closing bracket after 'how to search by reference'

File: input/intro-notes/StructureDefinition-au-core-procedure-notes.md [hunk 5]
```diff
@@ -41,19 +41,19 @@
         <td>code</td>
         <td><b>MAY</b></td>
         <td><code>token</code></td>
-        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.</td>
+        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.<br/><br/>The requester <strong>SHALL</strong> support <code>multipleOr</code>. The responder <strong>SHALL</strong> support <code>multipleOr</code>.</td>
```
Comment (dbojicic) [inline input/intro-notes/StructureDefinition-au-core-procedure-notes.md:44]: Should be '<strong>SHOULD</strong> support <code>multipleOr</code>'.

This change needs to be applied to AU Core Requester and Responder CapabilityStatements.

File: input/resources/au-core-requester.xml [hunk 6]
```diff
@@ -854,6 +854,7 @@
 <blockquote>
 <div>
 <p>If the requester supports the Encounter resource, the requester <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Encounter resource.</p>
+<p>The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both <code>Encounter.reasonCode</code> and <code>Encounter.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:857]: Apply formatting to Encounter.reasonCode and Encounter.reasonReference in the first sentence as well.

File: input/resources/au-core-requester.xml [hunk 7]
```diff
@@ -3194,7 +3201,7 @@
 </extension>
 <type value="Encounter"/>
 <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-encounter"/>
-<documentation value="If the requester supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource."/>
+<documentation value="If the requester supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource.&#xA;&#xA;The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both `Encounter.reasonCode` and `Encounter.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3204]: Apply formatting to all instances of Encounter.reasonCode and Encounter.reasonReference.

File: input/resources/au-core-responder.xml [hunk 8]
```diff
@@ -866,6 +866,7 @@
 <blockquote>
 <div>
 <p>If the responder supports the Encounter resource, the responder <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Encounter resource.</p>
+<p>The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both <code>Encounter.reasonCode</code> and <code>Encounter.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:869]: Apply formatting to all instances of Encounter.reasonCode and Encounter.reasonReference.

File: input/resources/au-core-responder.xml [hunk 9]
```diff
@@ -3205,7 +3213,7 @@
 </extension>
 <type value="Encounter"/>
 <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-encounter"/>
-<documentation value="If the responder supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource."/>
+<documentation value="If the responder supports the Encounter resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Encounter resource.&#xA;&#xA;The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both `Encounter.reasonCode` and `Encounter.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3216]: Apply formatting to all instances of Encounter.reasonCode and Encounter.reasonReference.

File: input/resources/au-core-requester.xml [hunk 10]
```diff
@@ -1504,6 +1505,7 @@
...
 <p>The MedicationRequest resources can represent a medication using either a code or refer to the Medication resource. When referencing Medication, the resource may be <a href="http://hl7.org/fhir/R4/references.html#contained">contained</a> or an external resource. The responder <strong>MAY</strong> choose any one way or more than one method, but if an external reference to Medication is used, the responder <strong>SHALL</strong> support the <code>_include</code> parameter for searching this element. The requester application <strong>SHALL</strong> support all methods.</p>
+<p>The MedicationRequest resource can communicate the clinical indication using either a code in MedicationRequest.reasonCode or a reference using MedicationRequest.reasonReference. Although both <code>MedicationRequest.reasonCode</code> and <code>MedicationRequest.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:1508]: Apply formatting to element names consistently.

File: input/resources/au-core-responder.xml [hunk 11]
```diff
@@ -1517,6 +1518,8 @@
...
+<p>The MedicationRequest resources can represent a medication using either a code or refer to the Medication resource. When referencing Medication, the resource may be <a href="http://hl7.org/fhir/R4/references.html#contained">contained</a> or an external resource. The responder <strong>MAY</strong> choose any one way or more than one method, but if an external reference to Medication is used, the responder <strong>SHALL</strong> support the <code>_include</code> parameter for searching this element. The requester application <strong>SHALL</strong> support all methods.</p>
+<p>The MedicationRequest resource can communicate the clinical indication using either a code in MedicationRequest.reasonCode or a reference using MedicationRequest.reasonReference. Although both <code>MedicationRequest.reasonCode</code> and <code>MedicationRequest.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:1522]: Apply formatting to element names consistently.

File: input/resources/au-core-requester.xml [hunk 12]
```diff
@@ -3644,7 +3651,7 @@
...
...application **SHALL** support all methods.&#xA;&#xA;If the requester supports the MedicationRequest resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the MedicationRequest resource.&#xA;&#xA;The MedicationRequest resources can represent a medication using either a code or refer to the Medication resource. When referencing Medication, the resource may be [contained](http://hl7.org/fhir/R4/references.html#contained)> or an external resource. The responder **MAY** choose any one way or more than one method, but if an external reference to Medication is used, the responder **SHALL** support the `_include` parameter for searching this element. The requester application **SHALL** support all methods.&#xA;&#xA;The MedicationRequest resource can communicate the clinical indication using either a code in MedicationRequest.reasonCode or a reference using MedicationRequest.reasonReference. Although both `MedicationRequest.reasonCode` and `MedicationRequest.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3654]: Apply formatting to element names consistently.

File: input/resources/au-core-responder.xml [hunk 13]
```diff
@@ -3651,7 +3659,7 @@
...
...application **SHALL** support all methods.&#xA;&#xA;If the requester supports the MedicationRequest resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the MedicationRequest resource.&#xA;&#xA;The MedicationRequest resources can represent a medication using either a code or refer to the Medication resource. When referencing Medication, the resource may be [contained](http://hl7.org/fhir/R4/references.html#contained)> or an external resource. The responder **MAY** choose any one way or more than one method, but if an external reference to Medication is used, the responder **SHALL** support the `_include` parameter for searching this element. The requester application **SHALL** support all methods.&#xA;&#xA;The MedicationRequest resource can communicate the clinical indication using either a code in MedicationRequest.reasonCode or a reference using MedicationRequest.reasonReference. Although both `MedicationRequest.reasonCode` and `MedicationRequest.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3662]: Apply formatting to element names consistently.

File: input/resources/au-core-requester.xml [hunk 14]
```diff
@@ -2675,6 +2681,7 @@
 <blockquote>
 <div>
 <p>If the requester supports the Procedure resource, the requester <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Procedure resource.</p>
+<p>The Procedure resource can communicate the clinical indication using either a code in Procedure.reasonCode or a reference using Procedure.reasonReference. Although both <code>Procedure.reasonCode</code> and <code>Procedure.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:2684]: Apply formatting to element names consistently.

File: input/resources/au-core-requester.xml [hunk 15]
```diff
@@ -4471,7 +4478,7 @@
 </extension>
 <type value="Procedure"/>
 <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-procedure"/>
-<documentation value="If the requester supports the Procedure resource, the requester **SHALL** support the AU Core profile and the conformance expectations for the Procedure resource."/>
+<documentation value="If the requester supports the Procedure resource, the requester **SHALL** support the AU Core profile and the conformance expectations for the Procedure resource.&#xA;&#xA;The Procedure resource can communicate the clinical indication using either a code in Procedure.reasonCode or a reference using Procedure.reasonReference. Although both `Procedure.reasonCode` and `Procedure.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:4481]: Apply formatting to element names consistently.

File: input/resources/au-core-responder.xml [hunk 16]
```diff
@@ -2689,6 +2696,7 @@
 <blockquote>
 <div>
 <p>If the responder supports the Procedure resource, the responder <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Procedure resource.</p>
+<p>The Procedure resource can communicate the clinical indication using either a code in Procedure.reasonCode or a reference using Procedure.reasonReference. Although both <code>Procedure.reasonCode</code> and <code>Procedure.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:2699]: Apply formatting to element names consistently.

File: input/resources/au-core-responder.xml [hunk 17]
```diff
@@ -4474,7 +4482,7 @@
 </extension>
 <type value="Procedure"/>
 <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-procedure"/>
-<documentation value="If the responder supports the Procedure resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Procedure resource."/>
+<documentation value="If the responder supports the Procedure resource, the responder **SHALL** support the AU Core profile and the conformance expectations for the Procedure resource.&#xA;&#xA;The Procedure resource can communicate the clinical indication using either a code in Procedure.reasonCode or a reference using Procedure.reasonReference. Although both `Procedure.reasonCode` and `Procedure.reasonReference` are marked as Must Support, a responder is not required to support both but **SHALL** support at least one of these elements. The requester**SHALL** support both elements."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:4485]: Apply formatting to element names consistently.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see inline comments

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md (hunk 2 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitioner-notes.md:44]: fixed.

File: input/intro-notes/StructureDefinition-au-core-practitioner-notes.md (hunk 3 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitioner-notes.md:58]: fixed

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md (hunk 4 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md:80]: fixed

File: input/resources/au-core-responder.xml (hunk 8 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml:869]: lol, don't need to repeat a statement on 'all instances of'

File: input/resources/au-core-requester.xml (hunk 10 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml:1508]: The absence of formatting in the beginning was intentional in all of these instances. But it's fine to change it.

File: input/resources/au-core-requester.xml (hunk 6 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml:857]: fixed in all instances.

File: input/intro-notes/StructureDefinition-au-core-procedure-notes.md (hunk 5 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-procedure-notes.md:44]: Procedure is SHALL for multipleOr on status. No change.

Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-procedure-notes.md:44]: fixed.

File: input/intro-notes/StructureDefinition-au-core-condition-notes.md (hunk 1 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-condition-notes.md:68]: fixed.

File: input/resources/au-core-responder.xml (hunk 8 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:869]: Fix not included in this PR.

File: input/resources/au-core-responder.xml (hunk 9 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:3216]: Fix not included in this PR.

File: input/resources/au-core-responder.xml (hunk 11 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:1522]: Fix not included in this PR.

File: input/resources/au-core-responder.xml (hunk 13 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:3662]: Fix not included in this PR.

File: input/resources/au-core-responder.xml (hunk 16 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:2699]: Fix not included in this PR.

File: input/resources/au-core-responder.xml (hunk 17 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:4485]: Fix not included in this PR.

**Review by dbojicic — CHANGES_REQUESTED** [review]
A few changes requested.

File: input/resources/au-core-responder.xml (hunk 8 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml:869]: I can see it in the PR.

Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed.

**Review by dbojicic — APPROVED** [review]

**Pull request merged**